src/luaclang.c

```diff
@@ -3,12 +3,146 @@
 #include "lualib.h"
 #include "lauxlib.h"
 
+#define INDEX  "ClangIndex"
```

Comment:
```suggestion
#define INDEX_METATABLE  "clang.Index"
```
I would follow this standard for all metatables.